api/tests/roundService.test.js

```diff
@@ -72,4 +73,26 @@ test('if end of player list, reset to 0', () => {
             score: 8,
             ID: 1
         });
-})
\ No newline at end of file
+})
+
+test('check if the user has won the game', () => {
```

Comment:
test names should say what they're expecting so it's more obvious when it fails whats gone wrong, so something like "If user has 10 points expect checkwin to return true"